lib/ansible/modules/windows/win_scheduled_task.py

```diff
@@ -57,7 +57,27 @@
       - absent
   user:
     description:
-      - User to run scheduled task as
+      - User to run the scheduled task as; defaults to the current user.  When only a user is set without specifying C(password) or
+        C(store_password), and the user is not a Windows built-in service account, the task is set to run only when the user
+        is logged in.
+    default: DOMAIN\user
+  password:
+    description:
+      - Password for the user account to run the scheduled task as
```

Comment:
Might be good to add, this is required if you wish to run the task without the user being logged in.